server/backend/database/memory/database.go

```diff
@@ -36,7 +36,8 @@ import (
 
 // DB is an in-memory database for testing or temporarily.
 type DB struct {
-	db *memdb.MemDB
+	db                      *memdb.MemDB
+	housekeepingProjectPage int
```

Comment:
DB instance is shared and used by several routines such as RPC handlers. Having a mutable value as a property in the instance seems to create other side effects.

How about moving `housekeepingProjectPage` into the housekeeping function?

Reply:
Thank you for the comment. I moved the variable into the run function.
Please let me know if I need to correct something!